### DISCUSSION TITLE
What is net vs gross consumption in Efficiency dashboard?

### DISCUSSION THREAD

**Discussion by SteveDonie** [teslamate-org/teslamate#2021] [Q&A]
The efficiency dashboard has two consumption numbers - net and gross. What is the difference between these two numbers?

**Comment by lq313c** [comment]
_Net_ is the efficiency while driving, while _gross_ is all consumption (including while idle, e.g. phantom drain, sentry mode, etc) over the period, as calculated by `range_loss`.

You can view the exact calculation by clicking on the dashboard title > Inspect > Query.

**Comment by SteveDonie** [comment]
Thanks’ I’ve been wondering for a long time.

**Comment by digodigo** [comment]
Hi, thanks for the information here. Yet, in my installation, _net_ ist always higher than _gross_. As for your explanation, shouldn't it be the other way round? I got two cars with the same data behaviour in teslamate.
